---
geometry: margin=1in

header-includes:
    - \usepackage{setspace}
    - \doublespacing
    - \usepackage{lineno}
    - \linenumbers
    - \usepackage{float}  # this controls the placement of images in PDF files (allows fig.pos = 'H')
    
# This section tells the rendering engine what we want as output. You can specify
# one format or many. The output below specifies three formats (PDF, HTML, Word)
# BUT by default only the first will be generated when you knit the document. To
# get around that (if you want all three for example) you can create a custom 
# function (or hook) that gets run when you press the knit button. 
# See "knit:..." below this.
output:
  pdf_document: 
    latex_engine: xelatex
    keep_tex: false
  html_document:
    keep_md: false
  word_document: default

csl: https://www.zotero.org/styles/elsevier-harvard

# this format for the bib files lets you include more than one. 
# The example given on the "grateful" gitHub page does not work for me. e.g.,
# bibliography: 
#   - my-library.bib
#   - grateful-refs.bib
# won't work
# if you are using Zotero then this line allows the references in these files to
# show up when you insert a ciation in the Visual editor. Also, the 
# grateful-refs.bib file is auto generated by the grateful package so you don't
# need to add it yourself.
bibliography: ["example_ms_my-library.bib"]

# this is a custom function that is called when you hit the "knit" button.
# it allows you to save your rendered document with a specific name (output_name)
# and location (output_dir). I have also set it up to render all formats at once
# as specified in the "output" part of the YAML header (above this) 

# output_name: -> the root name of the file(s) i.e., no extenstion
# output_path: -> this relative the location of Rmd file. In the case of this 
#                 example we are in the '04-manuscript' folder so 'rendered' is
#                 a subfolder of that.
knit: (
  function(inputFile, encoding) { 

    output_name <- 'mock_manuscript_JCL';
    folder_path <- 'rendered';  
                                
                                
  
    rmarkdown::render(
      input       = inputFile, 
      encoding    = encoding, 
      params      = list(sub_title = output_name),      
      output_file = output_name,
      output_dir = folder_path, output_format = "all") })
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
#loading libraries 
library("tidyverse")
library("tmap")
library(viridis)
library(sf)
```

**Title**: Anomalous latitudinal gradients in parasitoid wasp diversity - hotspots in regions with larger temperature range.

Jessica Castellanos-Labarcena^1^\*, Dirk Steinke^1,2^, Sarah J. Adamowicz1\^

1.  Department of Integrative Biology, University of Guelph, Guelph, Ontario, Canada
2.  Centre for Biodiversity Genomics, University of Guelph, Guelph, Ontario, Canada

\*Corresponding Author

email: jcaste01\@uoguelph.ca (JCL)

**Author Contributions:** JCL and SJA conceived the idea; JCL obtained and analyzed the data with assistance from SJA and DS; JCL led the writing with assistance from DS and SJA.

**Data Availability:** The data and code that support the findings of this study are openly available on GitHub at <https://github.com/Labarcena-Jessica/Assignment-Reproducibility>.

**Conflict of Interest statement**

No conflicts of interest

**Acknowledgements:** I would like to thank the instructors and all member of the Data Living Project for this excellent course.

\newpage

# Abstract

Parasitoid wasps are among the shortlist of taxa showing an anomalous latitudinal diversity gradient. Using the largest georeferenced molecular dataset, we used a macrogenetics approach to explore the spatial relationship between intra- and interspecific levels of diversity and potential environmental variables influencing the anomalous diversity pattern. Nucleotide diversity values were consistently higher at temperate latitudes, peaking at 50 degrees. We found a positive but weak relationship between intraspecific diversity and the latitude, between intra- and interspecific diversity, and a significant positive effect of the temperature range. Examining the spatial relationship between these levels of biodiversity and its drivers is particularly relevant considering climate change and its impact on species distribution. Yet, in insects, it has been challenging to integrate ecological, evolutionary, and geographical components when analyzing the processes leading to species richness gradients. Our study demonstrates how macrogenetics analyses of large-scale patterns of diversity provide insights into potential causes.

# Introduction

The emergence of comprehensive global datasets of the spatial distribution of species, as well as molecular and fossil record information, has facilitated global analyses of diversity gradients. However, most studies testing the latitudinal diversity gradient (LDG) have focused on vertebrates and woody plants, as those are the taxa with the most data available. A lack of data for most invertebrate groups has limited the number of analyses, although several pioneering studies have examined the LDG in insects [@condamine2012][@owen1974].

The most notable pattern observed is the one of species richness increasing from the poles to the equator [@saupe2019]. However, some groups, e.g., ichneumonid wasps, show an anomalous latitudinal diversity gradient, which is characterized by higher diversity in some temperate regions compared to tropical biomes [@owen1974]. These parasitoid wasps (Ichneumonoidea: Ichneumonidae, Braconidae and the recently described Trachypetidae family [@quicke2020] constitute a large and ecologically important group within the Hymenoptera, with approximately 41,000 described species, likely representing just a fraction of its true diversity. Ichneumonids play a key role in regulating populations of many groups of insects, including pests that affect agriculture, horticulture, and forestry. However, in particular, the tropical diversity of ichneumonoids is still poorly understood, as most sampling and species-level taxonomic descriptions have been focusing on temperate regions. Several studies highlight that the understanding of large-scale diversity patterns in these families is hindered by inadequate sampling of tropical species [@quicke2012].

Yet, this scenario is changing as large-scale studies are being conducted in Southeast Asia , South America, Uganda, and South Africa; revealing and characterizing the vast diversity of southern parasitoid wasps. Additionally, a new research program in Costa Rica aims to conduct a comprehensive, nationwide Malaise trapping program to barcode the nation's multicellular biodiversity. Nevertheless, based on current knowledge, these families show strong latitudinal patterns of species richness peaking at temperate latitudes and declining near the equator [@quicke2014].

This study used a macrogenetics approach to explore large-scale patterns and predictors of intraspecific genetic diversity in parasitoid wasps. To account for the taxonomic debt at lower latitudes, [@freeman2021] we used molecular clusters as a proxy for species, as in this highly diverse and less-studied group, a high number of cryptic species can be expected [@smith2008]. With very limited taxonomic expertise available worldwide, the use of DNA-based clustering of specimens into species has shown to be a low-cost and low-risk methodology in most taxa, especially in insects [@cariou2020]. This approach constitutes an area of active debate, so we implemented several quality control steps to account for taxonomic misclassification, contaminations, and sequencing errors within each molecular cluster. We used a publicly available genetic dataset for the DNA barcoding region cytochrome C oxidase subunit I (COI). Mitochondrial loci have long been used for exploring correlations between species and genetic diversity in global-scale macrogenetic studies, and despite the limitations associated with using a single gene, this is likely the best way to conduct large-scale biodiversity studies of largely unexplored taxonomic groups.

We also explored the effect of sampling coverage to determine the minimum number of sequences needed to calculate accurate intraspecific diversity estimates. The data were grouped at three different geographic scales to account for the fact that the geographic distance between pairs of conspecific sequences influences the expected genetic diversity in a set of DNA sequences. Therefore, we used grid cells, latitudinal bands, and climatic zones to explore global patterns of intraspecific genetic diversity. We examined the spatial relationship between intra- and interspecific levels of diversity and explored environmental predictor variables influencing the anomalous diversity pattern reported for this group.

# Methods

**Data Acquisition, filtering, and quality control**.

We downloaded the data directly into R (R Core Team, 2023) using the Barcode Of Life Data System (BOLD) [@ratnasingham2013] API (Application-Platform Interface) on October 19, 2021. The combined data set for Ichneumonidae and Braconidae consisted of 243,870 sequences of the 5' region of the cytochrome c oxidase subunit I gene (*COI*-5P), the barcode gene for animals.

**Intraspecific diversity estimation**

We calculated the nucleotide diversity per molecular cluster after grouping the data into three different spatial resolution levels: hexagonal grid cells, latitudinal bands, and climatic zones.  We used the R package dggridR (version 3.00) [@barnesandsahr2017] to build a global grid of hexagonal cells of 209,904 km^2^, all of which with identical sizes. The data were grouped by cells using the geographic coordinates associated with each record. We aligned the sequences of each molecular cluster in each cell using the R package muscle. The nucleotide diversity (Π) was calculated as the average number of variable sites in each pairwise sequence comparison using the R package PopGenome (verison 2.7.5) [@pfeifer2014]. Average pairwise nucleotide differences (using Nei's calculation of pi were divided by the number of base pairs to obtain diversity per site values. The function used, diversity.Stats, incorporates the calculated number of comparisons per site, which results in a Π value weighted by the number of comparisons. The average genetic diversity per cell was calculated as the mean of all nucleotide diversity values per molecular cluster in each cell. Considering that the intensity of sampling, both in terms of the number of molecular clusters per cell and the number of sequences in each molecular cluster, varied substantially among cells, we also evaluated the impact of this variation. Molecular clusters with 10 or more sequences were used for sensitivity tests following [@miraldoandreia2016]. These clusters and only cells with three or more molecular clusters were retained for further analysis. We mapped the nucleotide diversity values per cell using the R package tmap (version 3.3-3).

# Results

**Global pattern of intra- and interspecific diversity in parasitoid wasps**. The nucleotide diversity values for parasitoid wasps were found to be higher at temperate latitudes based on data analyzed at three geographical scales. We analyzed 85 cells and found that those with higher nucleotide diversity values were located between 40- and 60 degrees latitude (Figure 1).  We found a weak yet significant positive correlation between the absolute value of latitude and nucleotide diversity per cell (R^2^ = 0.094, p = 0.004).

```{r figure 1, echo=FALSE, message=FALSE, warning=FALSE, out.width="50%", fig.cap="Figure 1: Global patterns of nucleotide diversity for parasitoid wasps. (a) Hexagonal grid cells of approximately 209,904 km2 show the mean nucleotide diversity values in a colour gradient where yellow represents higher values. Using a dataset of 209,382 COI records grouped in 15,584 molecular clusters, nucleotide diversity (Π) per molecular cluster was calculated as the average number of variable sites in each pairwise sequence comparison. After filtering, 85 cells were retained for a minimum of three molecular clusters per cell and 10 COI sequences per molecular cluster."}


#reading file with the values of nucleotide diversity per cell. This is a shapefile that it is located in the 02_outdata folder. It contains the grid object with nucleotide diversity values per cell. 

sf_grid_more10 <- st_read ("sf_grid_mpre10.shp")

#Loading data for mapping

data("World")

world <- World %>% 
  filter(sovereignt != "Antarctica")

mapmore10.3_parasitoids <- tm_shape(world, projection="+proj=robin")+
  tm_polygons(col="grey",border.col= "grey")+
  tm_shape(sf_grid_more10, projection="+proj=robin") + tm_polygons(col = "mn___10",  palette = "viridis", style = "quantile", n = 6,border.col = NULL, border.alpha = 0.4, title = "Nucleotide Diversity")+
  tm_layout(legend.position = c("left","bottom"),
            legend.bg.color = "white", frame = FALSE)
mapmore10.3_parasitoids




```

# Discussion

Our results show that intra and interspecific diversity of parasitoid wasps covary and follow an atypical latitudinal diversity pattern with a peak at temperate latitudes. The weak but significant correlation between these two dimensions of biodiversity indicates common underlying evolutionary and ecological processes acting along a latitudinal gradient. While it has been previously noted that the latitudinal pattern of species richness for parasitoid wasps differs from the ones seen in other well-studied groups [@condamine2012][@lawrence], this study is the first to test the relationship between genetic diversity and a proxy for species richness. Our findings support the notion that the unexpected decline in diversity towards the tropics, which was previously identified through morphological data, is also evident in molecular data. The congruence between spatial variation of genetic diversity and species richness has been described for other groups of animals and was explained by well-supported hypotheses such as the evolutionary speed theory, the time and area, and the Red Queen hypothesis [@theodoridis2020]. Nevertheless, none of these can explain the anomalous pattern found for parasitoid wasps.

# References
